atlassian_jwt_auth/contrib/django/utils.py

```diff
@@ -7,7 +7,7 @@ def parse_jwt(verifier, encoded_jwt):
     claims = verifier.verify_jwt(
         encoded_jwt,
         settings.ASAP_VALID_AUDIENCE,
-        leeway=60
+        leeway=getattr(settings, 'ASAP_VALID_LEEWAY', 0)
```

Comment:
Seems like you're breaking the API here, so if that's the way you want to go with it, then api-break makes sense to me. That said, I'm not sure that a TTL of 0 seconds would ever make sense. Wouldn't it expire the moment it's created?

If I were doing it, I would probably add overriding the value as a feature and leave the default at 60 then later make an api-break to change the default if the need arises. I guess I'm not a big fan of breaking APIs.